### ISSUE TITLE
Associations Enhancements: A Collection of Github Tickets 

### ISSUE THREAD

**Issue by mei215** [spawaskar-cora/cora-docs#589]
There are several tickets in GitHub relating to a need for enhanced visibility of associations (pair matches, refits, articulations) that have been documented in Cora: 
#531 (Visibility of Whether there are Associations for an Element),
#453 (Overview of Articulations and Pair-Matching),
#486 (Search by Association),
#396 (mtDNA Report Enhancement), and more tangentially,
#116 and #567, regarding articulations reports.

The need for this visibility is increasing in the Ploesti Project (and other projects?), as more RoS are assigned, BECAUSE ALL PLOESTI DOCUMENTATION IS ENTIRELY in Cora (i.e., there are no additional Excel files of element types for searching/filtering elements).

So, for example, an analyst assigned a RoS by sequence must click on 1) "pair matches," 2) "refits", and 3) "articulations" for EACH element in the sequence to see if any associations have already been made.

Once the analyst has determined what elements can be added to a sequence by associations already documented in Cora, they must make lists of elements that must be checked for adding potential additional refits, pair matches, and articulations from other accessions. 

Currently available to assist the analyst with making these lists are the Articulations Report, Advanced Specimens Report, and Zone Report.

The Articulations Report only shows one step away articulations (e.g., a C2 vertebra will show it articulates with C3, but not if that C3 articulates through all the cervical, thoracic, lumbar, and sacral vertebrae.) Following the chain of articulations is a slow and inefficient process.

The Zones Report can be used to find potential refits, but the resulting list does not indicate if the elements in the list are already associated via PM, refit, artic to something else. Thus, the analyst must go through each one to see if it is a possibility for a refit. 

The Advanced Specimens Report can be used to start a list of pair matching elements, but again, there is no indication in the resulting list if the elements are already associated via PM, refit, artic to something else. Thus, the analyst must go through each one to see if it is a possibility for pair match. 

Then these lists made by the analyst are recreated by the peer reviewer of the RoS to make sure nothing was missed by the analyst. And sometimes these lists are recreated again by a project lead in a PL review. Overall, if associations were more visible, this could prevent mistakes/omissions and save time for analysts, peer reviewers, and project managers. 
 
Asks:

1. When an mtDNA, Zone, Advanced Specimens, or Articulations Report is generated, can there please be other columns visible that indicate a) what/if those remains have already been associated by articulation, refit, and pair match, and b) specifically to what Individual or Sequence? This would make filtering in a downloaded Excel much faster for us.

2. Ultimately, an Associations Report would capture all PM, refits, articulations of a mito sequence.

**Comment by fedamann** [comment]
@SachinPawaskarUNO @jwarsom Sounds like this would make a nice data visualization in order to see connections among specimens in one space, where 'relatedness' can be defined by method of association.

**Comment by TarawaProject** [comment]
I concur with Megan comments and feel that this is a needed analytics component.  Now that we have  data entered we need to be able to generate reports that improve our efficiency in comparisons.   If we are searching for a right femur to go with a left femur, we should not see those that have already been associated by other means to another left femur.  We should be able to cull via queries or the like.  Megan has captured what the Tarawa project does in separate excel sheets.

**Comment by clegarde** [comment]
This is super important for project management and analyst work flow, which is why we have discussed this on and off for a while. I have a note on the white board from our last discussion with @SachinPawaskarUNO and @mei215 in November/December on this issue when we discussed adding an Associations column(s) to report/search output. This would save a lot of time for analysts and project leads.

**Comment by mei215** [comment]
@fedamann @SachinPawaskarUNO @jwarsom @clegarde @skorps @TarawaProject @Katie-East @tlvandeest @veronicak7 can we please revisit this issue? It seems to me that the ability to see connections among specimens in one space in Cora (an Associations Report) is essentially the RoS. The existence of such a report would mean the RoS could essentially become paperless. Further, an exported Associations Report could easily be uploaded to LIMS--a way to start getting the CHR process into LIMS. Would love the chance to discuss and make this happen. This would make our current CHR process so much more streamlined and efficient

**Comment by SachinPawaskarUNO** [comment]
@fedamann I agree will @mei215. First this is inline with the conversation we have been having of being able to more closely integrate CoRA and LIMS/CMS either via data APIs or Exports/Reports. Second, we now actually have infrastructure in place to do exports of data (proof of concept was validated with Osteometric Sorting Pipeline). Finally, now we need to prioritize which custom data integration flows/reports we need to focus on and make it part of the next major deliverable.

**Comment by lfkennedy** [comment]
This ticket is related to visualizations requested by Audrey in managing the Myitkyina project (see ticket https://github.com/spawaskar-cora/cora-docs/issues/676). Can we revisit the build of this report/buildout @fedamann @SachinPawaskarUNO @mei215 @clegarde
